docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+1. Many developers do not read the theory and are just looking for the steps they need for a specific action. Bullets are an effective way to show this.
+2. Using unordered lists ensure all **actionable** steps are consistent throughout the docs and make the experience better for developers.
+3. Use ordered lists for explanations or theory in a list format.
+
+See the examples below:
+
+**Example #1**
+
+First, you will set-up a Nuxt.js project.
+
+- Install the **Vue CLI** from your terminal.
+
+```
+yarn global add @vue/cli
+
+# OR
+
+npm install -g @vue/cli
+
+```
+
+**Example #2**
+
+Getting started with a Node.js:
+
+1. You need to have a Node.js server installed
+2. You need to have Git Installed
+
+- Verify the version of Node.js:
+
+```bash
+node -v
+```
+
+- Verify the verion of Git:
+
+```bash
+
+git --version
+
+```
+
+---
+
+#### Indicate the path for commands
+
+Whenever you are asking someone to execute a command or to drop in a file, you indicate the `Path: ./` each time. For example, many tutorials involve going back and forth between the `backend` (Strapi) and `frontend` (Frontend Client), and individuals can get lost as to where to execute commands or install dependencies.
+
+See the examples below:
+
+**Example #1**
+
+- Create and enter a directory named `strapi-tutorial`:
+
+`Path: ./Projects/`
+
+```bash
+
+mkdir strapi-tutorial && cd strapi-tutorial
+```
+
+**Example #2**
+
+- Install the Strapi GraphQL plugin with the folllowing command:
+
+`Path:./strapi-tutorial/backend/`
+
+```bash
+cd backend
+
+yarn strapi install graphql
+
+# OR
+
+npm run strapi install graphql
+
+```
+
+Indicating the `Path ./` ensures that there are fewer possibilities for errors and misunderstanding.
+
+---
+
+#### Include or reference all steps necessary
+
+Each step necessary to accomplish the goal of the **Tutorial**, **How-to** or **Example** needs to be explained in detail or linked to an external resource that properly explains that particular step. When linking to an external resource, consider and explain which sections or steps to do.
+
+Please see the examples below:
+
+**Example #1**
+
+- Allow access to the **Roles & Permissions** section:
+  - (Public) Navigate to the `Roles & Permissions` in the left-hand menu of the Strapi Dashboard
+    - Click on the `Public` role
+    - Check the `find` and `findone` checkboxes of the `Dish` section
+    - Save
+
+**Example #2**
+
+- Install [Yarn](https://yarnpkg.com/en/) by following these instructions: [Yarn installation](https://yarnpkg.com/lang/en/docs/install/)
+
+Never assume steps are apparent. Please make your documentation clear to follow with no missing steps.
+
+---
+
+#### Emphasize concepts using "Note:"
+
+Occasionally, specific concepts or exceptions need to be pointed out _in context_ to the individual reading the documentation. The way to do this is using **Note:**. The markdown is as follows:
```

Comment:
I don't think we need to provide example on how to write markdown you already gave the link to the tutorial somewhere above. My thinking is that experienced developers will contribute to the documentation and more generally to Strapi so I think they are familiar with Markdown.

Reply:
I added the Markdown example as it's not easy to see that the colon **":"** is part of what is getting a bold treatment. Not because I am specifically trying to teach Markdown here. 

The only thing I changed here was to capitalize "Markdown".

Having said that when there is a push in the greater OSS community for new people to start contributing we can not be wrong by including examples like this. Articles like this https://www.firsttimersonly.com/ and for example, https://github.com/MunGell/awesome-for-beginners#javascript - this list, indicate a need for OSS projects to consider newer developers.